system/Database/Postgre/PreparedQuery.php

```diff
@@ -74,13 +74,11 @@ class PreparedQuery extends BasePreparedQuery implements PreparedQueryInterface
 	 * override this method.
 	 *
 	 * @param string $sql
-	 * @param array  $options Passed to the connection's prepare statement.
-	 *                        Unused in the MySQLi driver.
 	 *
 	 * @return mixed
 	 * @throws \Exception
 	 */
-	public function _prepare(string $sql, array $options = [])
```

Comment:
Since we are leaving it in for the base class we need it here to keep the signature the same.